### PR TITLE
Update doc to show max response data limit from 100 to 200 records + Add 400 bad request to GET api responses of expenses & merchants

### PR DESCRIPTION
**Pull request by modernwarfareuplink**

### CODE REVIEW

File: src/admin/paths/admin@expenses.yaml
Comment (modernwarfareuplink) [inline src/admin/paths/admin@expenses.yaml:25]: Created task to add 400 bad request as one of the response type to all GET apis once this is merged in. https://app.clickup.com/t/2at0ztt

File: src/admin/paths/admin@expenses.yaml
Comment (aditya999123) [inline src/admin/paths/admin@expenses.yaml:26]: what about approver and spender get calls ?

**Review by aditya999123 — APPROVED** [review]

**Review by aditya999123 — APPROVED** [review]

**Pull request merged**
